src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx

```diff
@@ -42,6 +42,15 @@ const AgendaItemRow = props => {
   const createDate = dateTernary(agenda?.creator_date);
   const updateByLast = agenda?.updaters?.last_name ? `${agenda.updaters.last_name},` : '';
   const updateDate = dateTernary(agenda?.modifier_date);
+  const formatScore = (score) => {
+    if (score === '--') return '-';
+    return score;
+  };
+
+  const formatDates = (currentDate) => {
```

Comment:
```suggestion
  const formatDate = (currentDate) => {
```

function expects a single date, so update to singular

Reply:
just looked at formatDate in utilities and it already handles returning a null if no date, so this function may be unnecessary

Reply:
Changes made for this

Reply:
Approved